governance/votes/Executive vote - July 27, 2022.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Template - [Executive Vote] Real World Assets Vault Onboardings - July 27, 2022
```

Comment:
Not sure if "Real World Asset Vault Onboardings" is better
Or even "Onboarding Real World Asset Vaults"

Reply:
Much better :)